---
layout: mypost
title: aweGNN with medical data science
categories: [文献速递]
---

### aweGNN with medical data science

#### packages & environment(most used)
- python
1. tensorflow 1/2
2. PyTorch 1.5.0
3. networkX
- R
1. igraph (graph data ,visualisation)
2. scapGNN (Active Pathway and Gene Module Inference from Single-cell Multi-omics Data) 
3. scGNNLTMG [BMEngineeR/scGNNLTMG: LTMG for scGNN (github.com)](https://github.com/BMEngineeR/scGNNLTMG)
4. cancerID_GNN [chopper6/cancerID_GNN: Biointelligence project: a graph neural network identifies cancer subtypes (github.com)](https://github.com/chopper6/cancerID_GNN)
5. Graph_PlusMinus [conormalone/Graph_PlusMinus: GNN based plus minus from NBA PBP Data (github.com)](https://github.com/conormalone/Graph_PlusMinus)
6. spaGCN package in R [JianingYao/spaGCN_R: spaGCN package in R (github.com)](https://github.com/JianingYao/spaGCN_R)
7. Flexible Protocol for Targeted Gene Co-expression Network Analysis [dcomanschmid/tGCN: Flexible Protocol for Targeted Gene Co-expression Network Analysis (github.com)](https://github.com/dcomanschmid/tGCN)
#### medical CV
1. 3dCT 分类 [iMED-Lab/UG-GAT (github.com)](https://github.com/iMED-Lab/UG-GAT)
#### medical Gene-gene/RNA/protein/metabolite interaction
1.   A graph-embedded deep feedforward network for disease outcome classification and feature selection using gene expression data [yunchuankong/GEDFN: GEDFN: Graph-Embedded Deep Feedforward Network (github.com)](https://github.com/yunchuankong/GEDFN)
2. `RESEPT` is a deep-learning framework for characterizing and visualizing tissue architecture from spatially resolved transcriptomics.[OSU-BMBL/RESEPT: A deep-learning framework for characterizing and visualizing tissue architecture from spatially resolved transcriptomics (github.com)](https://github.com/OSU-BMBL/RESEPT)
3. Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network [jianhuupenn/SpaGCN: SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network (github.com)](https://github.com/jianhuupenn/SpaGCN)
4. scFEA: A graph neural network model to estimate cell-wise metabolic using single cell RNA-seq data [changwn/scFEA: single cell Flux Estimation Analysis (scFEA) Try the below web server! (github.com)](https://github.com/changwn/scFEA)
5.   G-EDNN 数据处理important[2017100647/G-EDNN: Using expression quantitative trait loci data and graph-embedded neural networks to uncover genotype–phenotype interactions (github.com)](https://github.com/2017100647/G-EDNN)
6. NIHGCN:Predicting cancer drug response using parallel heterogeneous graph convolutional networks with neighborhood interactions [weiba/NIHGCN: Predicting cancer drug response using parallel heterogeneous graph convolutional networks with neighborhood interactions (github.com)](https://github.com/weiba/NIHGCN)
7. Prediction of gene co-expression from chromatin contacts with graph attention network[Page not found · GitHub](https://github.com/JieZheng-ShanghaiTech/HiCoEx.)
8. [wxbCaterpillar/GE-Impute (github.com)](https://github.com/wxbCaterpillar/GE-Impute)
9. A novel biomarker selection method combining graph neural network and gene relationships applied to microarray data[xwdshiwo/BioFSDatasets_and_code: code and datasets of ILRC (github.com)](https://github.com/xwdshiwo/BioFSDatasets_and_code)
10. Spatial Transcriptomics Prediction from Histology jointly through Transformer and Graph Neural Networks [biomed-AI/Hist2ST (github.com)](https://github.com/biomed-AI/Hist2ST)
11. Use of a graph neural network to the weighted gene co-expression network analysis of Korean native cattle [gywns6287/gmcNet: gene module clustering network (github.com)](https://github.com/gywns6287/gmcNet)
12.  GraphGONet: a self-explaining neural network encapsulating the Gene Ontology graph for phenotype prediction on gene expression[Victoria BOURGEAIS / GraphGONet · GitLab (univ-evry.fr)](https://forge.ibisc.univ-evry.fr/vbourgeais/GraphGONet)
13. # Prediction of Time Series Gene Expression and Structural Analysis of Gene Regulatory Networks Using Recurrent Neural Networks https://github.com/jonathan-f/DA_RNN_GENEXP
14.  Deep structural clustering for single-cell RNA-seq data jointly through autoencoder and graph neural network [DHUDBlab/scDSC (github.com)](https://github.com/DHUDBlab/scDSC)
15. Multi-level attention graph neural network based on co-expression gene modules for disease diagnosis and prognosis [github.com](https://github.com/TencentAILabHealthcare/MLA-GNN)
16. 【important】Identifying Cancer Subtypes Using a Residual Graph Convolution Model on a Sample Similarity Network [weiba/ERGCN at master (github.com)](https://github.com/weiba/ERGCN/tree/master)
17. Unsupervised construction of computational graphs for gene expression data with explicit structural inductive biases [paulmorio/gincco: Source code for GINCCo Paper (github.com)](https://github.com/paulmorio/gincco)
18. 【data】Distant metastasis identification based on optimized graph representation of gene interaction patterns [RanSuLab/Metastasis-glmGCN (github.com)](https://github.com/RanSuLab/Metastasis-glmGCN)
19. Detecting Spatially Co-expressed Gene Clusters with Functional Coherence by Graph-regularized Convolutional Neural Network [kuanglab/CNN-PReg (github.com)](https://github.com/kuanglab/CNN-PReg)
20. A Graph Feature Auto-Encoder for the prediction of unobserved node features on biological networks[RaminHasibi/GraphFeatureAutoencoder: A repo for implementation of Graph features autoencoder for expression values prediction and imputation (github.com)](https://github.com/RaminHasibi/GraphFeatureAutoencoder)
21. A graph convolutional neural network for gene expression data analysis with multiple gene networks
22. Improving cancer driver gene identification using multi-task learning on graph convolutional network [weiba/MTGCN (github.com)](https://github.com/weiba/MTGCN)